addons/io_hubs_addon/components/definitions/environment_settings.py

```diff
@@ -95,9 +95,15 @@ def draw(self, context, layout, panel):
             sub_row.enabled = False
             add_link_indicator(sub_row, self.backgroundTexture)
 
+        sub_row.context_pointer_set("hubs_component", self)
+        sub_row.context_pointer_set("host", self.backgroundTexture)
+        op = sub_row.operator("image.hubs_open_image", text='', icon='FILE_FOLDER')
```

Comment:
I missed this in the first review, but these should be on the `row` layout, not the `sub_row` layout.
Also, the host for this should be `context.scene`, not the image (we only need to disable stuff if the host and its component are linked)